src/ua/griddynamics/Game.java

```diff
@@ -0,0 +1,183 @@
+package ua.griddynamics;
+
+import java.util.Arrays;
+import java.util.Random;
+
+public class Game {
+    static final int WIDTH = 9;
+    static final int HEIGHT = 9;
+
+    public final char X = 'X';
+    public final char FREE = '/';
+    public final char MINE = '*';
+    public final char POINT = '.';
+
+    private final char[][] hiddenField;
+    private final char[][] visibleField;
+
+    public Game(char[][] hiddenField, char[][] visibleField) {
+        this.hiddenField = hiddenField;
+        this.visibleField = visibleField;
+    }
+
+    public void placeMines(Coordinate coordinate, int numberOfMines) {
+        Random random = new Random();
+        int minesPlaced = 0;
+        int userX = coordinate.getX();
+        int userY = coordinate.getY();
+        do {
+            int x = random.nextInt(WIDTH);
+            int y = random.nextInt(HEIGHT);
+            if (hiddenField[y][x] != X && x != userX || y != userY) {
```

Comment:
It is incorrect condition
Please take a look at https://www.programiz.com/java-programming/operator-precedence

Reply:
Done